assets/eip-5851/contracts/ERC5851Issuer.sol

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
Please make the EIP-3475 changes in a separate PR.

Reply:
Hi @Pandapip1 can you approve this version,now ERC-3475 file stay unchanged.